trestle/tasks/csv_to_oscal_cd.py

```diff
@@ -857,6 +883,72 @@ def get_props(self) -> List[Property]:
         return rval
 
 
+class _ResolvedProfileCatalogHelper():
+    """Resolved Profile Catalog Helper."""
+
+    def __init__(self, profile_list: List[str], root: str = '.') -> None:
+        """Initialize."""
+        self._profile_list = profile_list
+        self._root = root
+        self._init = False
+
+    def _initilaize(self):
+        if not self._init:
+            self._profile_map = {}
+            self._control_list = []
```

Comment:
initialize is spelled wrong - and this routine could be handled by creating a catalog_interface for the catalog, and then looping over all controls to get the parts.  catalog_interface provides a flat view of all controls that you can access by id.  Note also that controls may be "free" and not in a group - and the catalog_interface handles that.

Reply:
Duh. Fixed.